src/main/java/picard/sam/ReplaceSamHeader.java

```diff
@@ -53,8 +53,8 @@
         programGroup = SamOrBam.class
 )
 public class ReplaceSamHeader extends CommandLineProgram {
-    static final String USAGE_SUMMARY = "Replaces the SAMFileHeader in a SAM or BAM file.  ";
-    static final String USAGE_DETAILS = "This tool makes it possible to replace the header of a SAM or BAM file with the header of another" +
+    static final String USAGE_SUMMARY = "Replaces the SAMFileHeader in a SAM, BAM or CRAM file.  ";
+    static final String USAGE_DETAILS = "This tool makes it possible to replace the header of a SAM, BAM or CRAM  file with the header of another" +
             "file, or a header block that has been edited manually (in a stub SAM file). The sort order (@SO) of the two input files must " +
             "be the same.<br /><br />" +
             "Note that validation is minimal, so it is up to the user to ensure that all the elements referred to in the SAMRecords " +
```

Comment:
The CRAM format specifies that each container in a CRAM stream contain a [value](https://github.com/samtools/htsjdk/blob/fbba5364e1809de071bc479f30e4e2c8b17f5bbe/src/main/java/htsjdk/samtools/cram/structure/Container.java#L60) indicating that container's offset with its containing stream (!). I think there maybe one for each slice as well. If resetting the header displaces the rest of the stream, these values will be off. AFAIK, htsjdk doesn't actually use these values for anything currently, but it does set them. So generic block-type moves are probably not a good idea unless we implement a generalized method that knows how and when to repair these (which BTW we could use for other things).